<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>160</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="btnTriangle_2">
       <property name="text">
        <string>Triangle</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnSquare_2">
       <property name="text">
        <string>Square</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btnTrapezium_2">
       <property name="text">
        <string>Trapezium </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnPoly_2">
       <property name="text">
        <string>Polygon</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="tlImageView">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>641</width>
      <height>291</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeleteClass">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lwImages">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClasses">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Classes...</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lwClasses">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>75</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="tvFileName_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Classes</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>540</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Class file</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>75</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnImages">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Images...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeleteImage">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDispalyImg">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Display</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>641</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1218</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
